### PR TITLE
Bug fixes related to Zimbra integration.

### PR DESCRIPTION
**Pull request by BrantOlsen**
The alarm bug fix is the biggest functionality change as it allows recurring event exceptions to actually be read from the Zimbra server.

### CODE REVIEW

File: CalDav/Event.cs [hunk 1]
```diff
@@ -88,10 +89,30 @@ ICollection<IAlarm> IEvent.Alarms
 					case "CLASS": Class = value.ToEnum<Classes>(); break;
 					case "CREATED": Created = value.ToDateTime(); break;
 					case "DESCRIPTION": Description = value; break;
-					case "DTEND": End = value.ToDateTime(); break;
-					case "DTSTAMP": DTSTAMP = value.ToDateTime().GetValueOrDefault(); break;
-					case "DTSTART": Start = value.ToDateTime(); break;
-					case "LAST-MODIFIED": LastModified = value.ToDateTime(); break;
+                    case "DTEND":
```
Comment (altostratous) [inline CalDav/Event.cs]: Can you please kindly fix this indentation mismatch?

File: CalCli.API/AlarmActions.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@ namespace CalCli.API
 {
     public enum AlarmActions
     {
+        NONE,
```
Comment (altostratous) [inline CalCli.API/AlarmActions.cs:10]: To my knowledge, the behavior defined in [CalDav RFC](https://tools.ietf.org/html/rfc4791#section-8.6), has nothing to tell about the `NONE` action. With this change, we are implementing something more tolerant than the API itself.

Can you please provide documentation on how recurring alarms or events are stored or the reason to parse `NONE` as a valid action?

**Review by altostratous — CHANGES_REQUESTED** [review]
I kindly appreciate your PR. Please check my review. I'll propagate your PR upwards whenever you and I agree that everything is OK.

File: CalCli.API/AlarmActions.cs (hunk 2 above)
Comment (BrantOlsen) [reply CalCli.API/AlarmActions.cs:10]: Zimbra returns the following where there is a reminder set to 5 minutes before the event.

    BEGIN:VALARM
    ACTION:NONE
    TRIGGER;VALUE=DATE-TIME:19760401T005545Z
    DESCRIPTION:Reminder
    X-WR-ALARMUID:1C414420-2E9C-47D0-9084-9E110714D53A
    END:VALARM
    BEGIN:VALARM
    ACTION:DISPLAY
    TRIGGER:-PT5M
    DESCRIPTION:Reminder
    X-WR-ALARMUID:4DFA4853-A1FA-46AD-8BF1-C706821FB79F
    END:VALARM

I have no idea why Zimbra does this just that it does.

**Review by altostratous — APPROVED** [review]

**Comment by altostratous** [comment]
OK, I hope this tolerance won't have bad effects...

**Pull request merged**